src/gtnh/models/mod_version_info.py

```diff
@@ -0,0 +1,15 @@
+from typing import Optional
+
+from gtnh.defs import Side
+from gtnh.models.base import GTNHBaseModel
+
+
+class ModVersionInfo(GTNHBaseModel):
```

Comment:
I'd add a create() method that takes a ModInfo, pulls out what it needs in one place, rather than scattered throughout the code.  

Not necessarily needed now, I can switch it up later if the number of fields expands.